lib/manageiq/session.rb

```diff
@@ -0,0 +1,64 @@
+require "patches/session_patch"
```

Comment:
Note:  I moved this file (at the bottom of the "Files changed", but doesn't show as a diff).

Was just a patch as well, so moved it and it this file will load it all the same when this class is loaded by  `config/initializers/session_store.rb`